pkg/server/handler.go

```diff
@@ -138,16 +138,16 @@ func (mc *MsgHandler) handleClientMessages(msg network.InMsg, token string) erro
 }
 
 func (mc *MsgHandler) handleRequestReplyCommands(msg network.InMsg, token string) error {
-	corrID, ok := msg.Headers["correlation-id"].(string)
+	replyTo, ok := msg.Headers["reply_to"].(string)
 	if !ok {
-		return errors.New("correlation ID not provided")
+		return errors.New("reply_to property not provided")
 	}
```

Comment:
I think it should be added and not replaced

Reply:
You should pass both (correlationID and reply_to) otherwise the client won't know if that is the reply to its request

Reply:
Actually, it depends on the client's needs. The way we implement them doesn't require us to use the `correlation_id` for now. As discussed with @vitbaq, the message broker doesn't enforce any rule about the RPC paradigm and we're free to choose how to use those properties.

However, there's no problem in passing the `correlation_id` too so that other scenarios could be covered without further changes. So, your suggestion will be considered and I should update the code soon.

Reply:
Done.